crates/sui-graphql-rpc/src/types/gas.rs

```diff
@@ -111,6 +113,7 @@ impl GasCostSummary {
     }
 }
 
+/// The gas effects (cost summary, gas object id, etc.)
```

Comment:
since this is basically repeating the name of the type im not sure we need a description here

Reply:
```suggestion
/// Effects related to gas (costs incurred and the identity of the smashed gas object returned).
```